docs/websites/nginx/nginx-ssl-and-tls-deployment-best-practices.md

```diff
@@ -0,0 +1,286 @@
+
+        systemctl restart nginx
+
+3.  In a web browser, navigate to the [Qualys SSL Labs SSL Server Test](https://www.ssllabs.com/ssltest/). Enter the domain name or hostname of your Linode and click "Submit". Optionally, you may uncheck the checkbox to prevent your test from being shown publicly.
+
+    Once the test is complete, scroll down to the "Protocol Details" section. Look for the "OCSP stapling" line. If nginx is configured correctly, this test will return "Yes".
+
+    [![SSL Server Test OCSP](/docs/assets/OCSP_Stapling_SSL_Test.jpg)](/docs/assets/OCSP_Stapling_SSL_Test.jpg)
+
+## HTTP Strict Transport Security (HSTS)
+
+Google Chrome, Mozilla Firefox, Opera, and Safari currently honor HSTS headers. HSTS is used to force browsers to only connect using secure encrypted connections. This means your site will no longer be accessible over HTTP. When HSTS is enabled and a valid HSTS header is stored in a users browser cache, the user will be unable to access your site if presented with a self-signed, expired, or SSL certificate issued by an untrusted certificate authority. The user will also be unable to bypass any certificate warnings unless your HSTS header expires or the browser cache is cleared.
+
+With all traffic being redirected from HTTP to HTTPS, you may want to allow users to only connect using HTTPS. Before enabling HSTS, be sure that you understand the potential impact on compatibility with older browsers.
+
+**Do not follow these steps if you want users to be able to access your site over HTTP!**
+
+1.  Open up your nginx HTTPS virtual host configuration file. This may be located at `/etc/nginx/sites-enabled/default` or at `/etc/nginx/conf.d/example_ssl.conf`. Append the following line inside your `server` block: 
+
+    {: .file-excerpt}
+    /etc/nginx/conf.d/example_ssl.conf
+    :   ~~~ conf    
+        add_header Strict-Transport-Security "max-age=31536000; includeSubdomains";
+        ~~~
+
+    The `max-age` attribute sets the expiration date for this header in seconds; in the above configuration, the header will expire after 1 year. You can configure this to be longer or shorter if you choose, but a period of less than 180 days is considered too short for the Qualys test. The `includeSubdomains` argument enforces HSTS on all subdomains.
+
+2.  Save your changes and restart nginx.
+
+        systemctl restart nginx
+
+3.  Navigate to the [Qualys SSL Labs SSL Server Test](https://www.ssllabs.com/ssltest/). Enter the domain name or hostname of your Linode and click "Submit". Optionally, you may uncheck the checkbox to not show your results on the boards.
+
+    {: .note}
+    >
+    > If you've already conducted a test from one of the above sections, use the **Clear cache** link to initiate a new scan.
+
+    Once the test is complete, scroll down to the "Protocol Details" section. Look for the "Strict Transport Security (HSTS)" line. If nginx is configured correctly this test will return "Yes".
```

Comment:
Proof note: "Yes". = "Yes." (same as above; these are system generated messages - not commands to type -  so proper punctuation mechanics should be used.)